tests/client/amo/sagas/testIndex.js

```diff
@@ -0,0 +1,15 @@
+import { fork } from 'redux-saga/effects';
+
+import rootSagas from 'amo/sagas';
+import categoriesSaga from 'amo/sagas/categories';
+
+
+describe('amo rootSagas', () => {
+  it('should get Api from state then make API request to categories', () => {
+    const sagaGenerator = rootSagas();
+
+    assert.deepEqual(sagaGenerator.next().value, [
+      fork(categoriesSaga),
```

Comment:
I know, the current test we have for registering reducers feels a bit overkill for me. I get how we want to add coverage to this code but other than that it feels like a duplication of effort.